.github/workflows/update-ic-did.yml

```diff
@@ -0,0 +1,63 @@
+name: Update ic.did
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # Runs at UTC midnight every day
+
+env:
+  # When getting Rust dependencies, retry on network error:
+  CARGO_NET_RETRY: 10
+  # Use the local .curlrc
+  CURL_HOME: .
+  # Disable DFX telemetry
+  DFX_TELEMETRY: 'off'
+  # Path to the ic.did file in this repository
+  FILE_PATH: "src/dfx/src/util/ic.did"
+
+jobs:
+  check-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout dfx repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download latest ic.did
+        id: get-file
+        run: |
+          # Download latest version from portal repo
+          EXTERNAL_URL="https://raw.githubusercontent.com/dfinity/portal/refs/heads/master/docs/references/_attachments/ic.did"
```

Comment:
Using master here would try to remove the VetKd declarations, see the latest PRs opened. I used the VetKd branch to sync the ic.did file in my recent [PR](https://github.com/dfinity/sdk/pull/4171). CC @lwshang